packages/template-retail-react-app/app/components/recommended-products/index.jsx

```diff
@@ -198,7 +198,7 @@ RecommendedProducts.propTypes = {
     recommender: PropTypes.string,
 
     /* The product IDs to use for recommendation context */
-    products: PropTypes.arrayOf(PropTypes.string),
+    products: PropTypes.arrayOf(PropTypes.object),
```

Comment:
This is not actually part of the needed changes but an existing error. I just fixed it to have a clean console to verify that pwa-kit is behaving as expected